README.md

```diff
@@ -4,43 +4,57 @@ This Looker Open Source repository is released under the MIT license. By using t
 
 While Looker has developed and tested this code internally, we cannot guarantee that the open-source tools used by the scripts in this repository have not been modified with malicious code.
 
-Our goal is to help people who want use Looker as a platform to get up and running quickly, largely by providing pre-built client SDKs in the most popular languages, and implementing consistency across all languages and platforms.
+## Overview
 
-The Looker API is defined with the [OpenAPI specification](https://github.com/OAI/OpenAPI-Specification), formerly known as "swagger." This specification is used to produce both Looker's interactive API Explorer,and the Looker API language bindings that describes the Looker REST API.
+This repository contains:
+
+- The [SDK code generator](packages/sdk-codegen) that generates the source code for Looker SDKs
```

Comment:
packages/sdk-codegen/README.md has a reference back to the root that seems pointless, maybe just remove it? or maybe it should have the `#looker-sdks` anchor?